ch04.adoc

```diff
@@ -210,96 +227,103 @@ double time(time) ;
 ====
 
 
-[[time-coordinate-units, Section 4.4.1, "Time Coordinate Units"]]
+[[time-coordinate-units, Section 4.4.2, "Time Coordinate Units"]]
 ==== Time Coordinate Units
 
 The **`units`** attribute of a time coordinate variable takes a string value that follows the formatting requirements of the <<UDUNITS>> package (e.g. <<time-axis-ex>>).
 It must comprise a unit of measure that is physically equivalent to the SI base unit of time (i.e. the second), followed by the word **`since`** and a reference datetime.
-The format of the **`units`** string implies that the time coordinate equals the length of the time interval from the instant identified by the reference datetime to the instant identified by the time coordinate.
-This is exactly true in all cases except when leap seconds occur between the two intervals in the **`standard`**, **`proleptic_gregorian`**, and **`julian`** calendars.
+By a "datetime" we mean the time and date together, specified as a set of six numbers (year, month, day, hour, minute, second), which are integers except for the second, which may have a fraction.
+The time coordinate exactly equals the length of the time interval from the instant identified by the reference datetime to the instant identified by the time coordinate, in all cases except when leap seconds occur between the two intervals in the **`standard`** calendar.
 See <<leap-seconds>>.
 
-The acceptable units of measure for time are given by UDUNITS.
-The most commonly used of these strings (and their abbreviations) are **`day`** (**`d`**), **`hour`** (**`hr`**, **`h`**), **`minute`** (**`min`**) and **`second`** (**`sec`**, **`s`**).
+The CF standard follows UDUNITS (<<units>>) in the definition of the acceptable
+units of measure for time.
+The most commonly used of these units strings (and their abbreviations) are **`day`** (**`d`**), **`hour`** (**`hr`**, **`h`**), **`minute`** (**`min`**) and **`second`** (**`sec`**, **`s`**).
 Plural forms are also acceptable.
+Any unit may optionally have one of the decimal prefixes for multiples and submultiples (<<table-supported-units, Table 3.1>>), but these are uncommon except for submultiples of **`second`** e.g. **`millisecond`** or **`ms`**.
 
 UDUNITS defines a **`year`** to be exactly 365.242198781 days (the interval between 2 successive passages of the sun through vernal equinox).
 __It is not a calendar year.__ UDUNITS defines a  **`month`** to be exactly **`year/12`**, which is __not a calendar month__.
-The CF standard follows UDUNITS in the definition of units, but we recommend that **`year`** and **`month`** should not be used, because of the potential for mistakes and confusion.
+We recommend that **`year`** and **`month`** should not be used, because of the potential for mistakes and confusion.
 
 UDUNITS defines a **`minute`** as 60 **`seconds`**, an **`hour`** as 3600 **`seconds`** and a **`day`** as 86400 **`seconds`**.
 These are not calendar units.
```

Comment:
Following the suggested change to the previous line I suggest to delete this line. From the context here it is not absolutely clear what a "calendar unit" is, and when/where they might be different from what is common knowledge. The previous line provide all the precision that is necessary at this point in the text.

Reply:
I've deleted "These are not calendar units", and moved the "fixed units of measure" phrase, which we already had, to replace it.